test/YesSql.Tests/project.json

```diff
@@ -19,7 +19,7 @@
       "dependencies": {
         "NETStandard.Library": "1.5.0-*",
         "dotnet-test-xunit": "1.0.0-dev-*",
```

Comment:
I removed these lines in the YesSql.Tests project.json file.

```
    "net451": {
      "frameworkAssemblies": {
        "System.Runtime": {
          "type": "build"
        },
        "System.Reflection": {
          "type": "build"
        },
        "System.Threading.Tasks": {
          "type": "build"
        },
        "System.Runtime.Serialization": "4.0.0.0"
      },
      "dependencies": {
      }
    }
```
And ran tests succesfully : 

![image](https://cloud.githubusercontent.com/assets/3228637/15151896/33b66386-16a1-11e6-992a-227829769924.png)
![image](https://cloud.githubusercontent.com/assets/3228637/15151959/769c177c-16a1-11e6-9e0e-9dbd6ba91bac.png)


So this is related to : https://github.com/aspnet/KoreBuild/pull/64